packages/sil-voting-app/src/routes/dashboard/components/Proposal/RenderedProposal.tsx

```diff
@@ -0,0 +1,24 @@
+import Hairline from 'medulas-react-components/lib/components/Hairline';
+import React from 'react';
+
+import Proposal, { ProposalProps } from '.';
+
+interface Props {
+  readonly proposal: ProposalProps;
+  readonly index: number;
+}
+
+const RenderedProposal = (props: Props): JSX.Element => {
+  if (props.index === 0) {
+    return <React.Fragment key={props.index}>{Proposal(props.proposal)}</React.Fragment>;
```

Comment:
You can set `key` for `<RenderedProposal key={index}/>` instead of passing it to the component itself. Also using array `index` as `key` propery is not good idea. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
I was aware, totally forgot to change it when I had `proposal.id` in scope, thank you.